internal/commands/util/pr.go

```diff
@@ -0,0 +1,99 @@
+	"github.com/checkmarx/ast-cli/internal/wrappers"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func NewPRDecorationCommand(prWrapper wrappers.PRWrapper) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "pr",
+		Short: "Posts the comment with scan results on the Pull Request",
+		Example: heredoc.Doc(
+			`
+			$ cx utils pr <scm-name>
+		`,
+		),
+	}
+
+	prDecorationGithub := PRDecorationGithub(prWrapper)
+
+	cmd.AddCommand(prDecorationGithub)
+	return cmd
+}
+
+func PRDecorationGithub(prWrapper wrappers.PRWrapper) *cobra.Command {
+	prDecorationGithub := &cobra.Command{
+		Use:   "github",
+		Short: "Decorate github PR with vulnerabilities",
+		Long:  "Decorate github PR with vulnerabilities",
+		Example: heredoc.Doc(
+			`
+			$ cx utils pr github --scan-id <scan-id> --token <PAT> --namespace <organization> --repo-name <repository>
+                --pr-number <pr number>
+		`,
+		),
+		Annotations: map[string]string{
+			"command:doc": heredoc.Doc(
+				`	https://checkmarx.atlassian.net/l/cp/tydFqUhE
+			`,
+			),
+		},
+		RunE: runPRDecoration(prWrapper),
+	}
+
+	prDecorationGithub.Flags().String(params.ScanIDFlag, "", "Scan ID is needed")
+	prDecorationGithub.Flags().String(params.SCMTokenFlag, "", params.GithubTokenUsage)
+	prDecorationGithub.Flags().String(params.NamespaceFlag, "", params.NamespaceFlagUsage)
+	prDecorationGithub.Flags().String(params.RepoNameFlag, "", params.RepoNameFlagUsage)
+	prDecorationGithub.Flags().Int(params.PRNumberFlag, 0, params.PRNumberFlagUsage)
+
+	// Set the value for token to mask the scm token
+	_ = viper.BindPFlag(params.SCMTokenFlag, prDecorationGithub.Flags().Lookup(params.SCMTokenFlag))
+
+	// mark all fields as required\
+	_ = prDecorationGithub.MarkFlagRequired(params.ScanIDFlag)
+	_ = prDecorationGithub.MarkFlagRequired(params.SCMTokenFlag)
+	_ = prDecorationGithub.MarkFlagRequired(params.NamespaceFlag)
+	_ = prDecorationGithub.MarkFlagRequired(params.RepoNameFlag)
+	_ = prDecorationGithub.MarkFlagRequired(params.PRNumberFlag)
+
+	return prDecorationGithub
+}
+
+func runPRDecoration(prWrapper wrappers.PRWrapper) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		scanID, _ := cmd.Flags().GetString(params.ScanIDFlag)
+		scmTokenFlag, _ := cmd.Flags().GetString(params.SCMTokenFlag)
+		namespaceFlag, _ := cmd.Flags().GetString(params.NamespaceFlag)
+		repoNameFlag, _ := cmd.Flags().GetString(params.RepoNameFlag)
+		prNumberFlag, _ := cmd.Flags().GetInt(params.PRNumberFlag)
+
+		prModel := &wrappers.PRModel{
+			ScanID:    scanID,
+			ScmToken:  scmTokenFlag,
+			Namespace: namespaceFlag,
+			RepoName:  repoNameFlag,
+			PrNumber:  prNumberFlag,
+		}
+
+		prResponse, errorModel, err := prWrapper.PostPRDecoration(prModel)
+
+		if err != nil {
+			return err
+		}
+
+		if errorModel != nil {
+			return errors.Errorf("Failed creating PR Decoration")
+		}
+
+		logger.Print(prResponse)
```

Comment:
Should we print this in debug mode?

Reply:
It is just printing the message from the HTTP rest, i think we should print always to provide feedback to the user.